doc/Extensions/Applications.md

```diff
@@ -137,6 +138,22 @@ extend bind /etc/snmp/bind
 
 3: Set the variable 'agent' to '1' in the config.
 
+### C.H.I.P
+
+C.H.I.P. is a $9 R8 based tiny computer ideal for small projects.  
+Further details: https://getchip.com/pages/chip
+
+#### SNMP Extend
+1. Copy the shell script to the desired host (the host must be added to LibreNMS devices)
```

Comment:
You should include the command to download the file as per other application docs.